cmd/helm/completion.go

```diff
@@ -244,3 +247,7 @@ __helm_bash_source <(__helm_convert_bash_to_zsh)
 	out.Write([]byte(zshTail))
 	return nil
 }
+
+func runCompletionFish(out io.Writer, cmd *cobra.Command) error {
+	return cmd.GenFishCompletion(out, !completionNoDesc)
```

Comment:
Actually, this does not work.  You need to call `cmd.Root().GenFishCompletion(...)`.